src/NuGet.Clients/NuGet.SolutionRestoreManager/RestoreOperationLogger.cs

```diff
@@ -275,18 +275,7 @@ public bool ShouldShowMessageAsOutput(MSBuildVerbosityLevel verbosity)
 
         public async Task LogExceptionAsync(Exception ex)
         {
-            string message;
-            if (OutputVerbosity < 3)
-            {
-                message = string.Format(CultureInfo.CurrentCulture,
-                    Resources.ErrorOccurredRestoringPackages,
-                    ex.Message);
-            }
-            else
-            {
-                // output exception detail when _msBuildOutputVerbosity is >= Detailed.
-                message = string.Format(CultureInfo.CurrentCulture, Resources.ErrorOccurredRestoringPackages, ex);
-            }
+            string message = string.Format(CultureInfo.CurrentCulture, Resources.ErrorOccurredRestoringPackages, ex);
```

Comment:
My only concern is how this works with inaccessible feeds. 
In particular, I think exceptions is how those failures are propagated, which is not ideal.

Reply:
then that should be an explicit case that writes a different message

Reply:
I agree, which is I mentioned that the current behavior is not ideal, 

We should understand if we can change that behavior first before logging the stack trace for every exception.